awx/ui_next/src/components/JobList/JobListItem.jsx

```diff
@@ -63,24 +71,26 @@ function JobListItem({
             </DataListCell>,
           ]}
         />
-        {job.type !== 'system_job' &&
-          job.summary_fields?.user_capabilities?.start && (
-            <DataListAction
-              aria-label="actions"
-              aria-labelledby={labelId}
-              id={labelId}
-            >
-              <Tooltip content={i18n._(t`Relaunch Job`)} position="top">
-                <LaunchButton resource={job}>
-                  {({ handleRelaunch }) => (
-                    <Button variant="plain" onClick={handleRelaunch}>
-                      <RocketIcon />
-                    </Button>
-                  )}
-                </LaunchButton>
-              </Tooltip>
-            </DataListAction>
+        <DataListAction
```

Comment:
The DataListAction component requires children, so we need to check `job.type !== 'system_job' &&
            >	          job.summary_fields?.user_capabilities?.start` before we render the DataListAction component.